runtime/README.md

```diff
@@ -0,0 +1,10 @@
+Minimal Runtime for the Planner.
+
+To use this with the Planner library:
+
+0. Get dependencies for Planner: `npm install`
+1. Build the Planner library: `gulp build:library`
+2. Go to the library build directory and link to npm cache: `npm link`
```

Comment:
Please explain what "go to the library build directory" means. On point 3. you also explain `cd runtime`.